packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -1,28 +1,24 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+%global gem_name katello
+%global plugin_name katello
 %global foreman_min_version 3.7
 %global foreman_max_version 3.8
-%global plugin_name katello
-%global gem_name katello
 %global prereleasesource pre.master
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
 %global mainver 4.9.0
-%global release 1
-
-Name:    %{?scl_prefix}rubygem-%{gem_name}
-Summary: Content and Subscription Management plugin for Foreman
+%global release 2
 
+Name: rubygem-%{gem_name}
 Version: %{mainver}
 Release: %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
-Group:   Applications/Systems
-License: GPLv2
-URL:     https://theforeman.org/plugins/katello
+Summary: Content and Subscription Management plugin for Foreman
+License: GPL-2.0
```

Comment:
Isn't `GPLv2` the correct Fedora tag? I'm not sure we can use SPDX codes in EL8 (though Fedora has adopted them).

Reply:
maybe. https://github.com/theforeman/foreman-packaging/blob/rpm/develop/licenses.json doesn't list the "no later" version, so probably that's why it got reset to the value in https://github.com/Katello/katello/blob/2cad75d2d309afe0061f2f656d80409feff6c180/katello.gemspec#L10